### ISSUE TITLE
Basic authentication using a password to the API is deprecated and will soon no longer work

### ISSUE THREAD

**Issue by juananruiz** [pgogy/Wordpress-Github-Widget#19]
You used a password to access an endpoint through the GitHub API using curl/7.38.0:

https://api.github.com/user/211510/repos

Basic authentication using a password to the API is deprecated and will soon no longer work. Visit https://developer.github.com/changes/2020-02-14-deprecating-password-auth/ for more information around suggested workarounds and removal dates.

Thanks,
The GitHub Team

**Comment by pgogy** [comment]
Yeah, that's been a thing for a year or so, will try to work a fix in too
________________________________

**Comment by GwynethLlewelyn** [comment]
Mmh and have you? :-)

**Comment by pgogy** [comment]
Not yet no On 2 Jul 2023, at 16:19, Gwyneth Llewelyn ***@***.***> wrote:﻿
Mmh and have you? :-)
